packaging/debian/dotnet-nightly-debian_config.json

```diff
@@ -0,0 +1,54 @@
+{
+    "maintainer_name":"Microsoft",
+    "maintainer_email": "dotnetcore@microsoft.com",        
+
+    "package_name": "dotnet-nightly",                           
```

Comment:
Talked offline.  -dev  is a commonly used convention in debian for shipping header files and libraries.

We will stick with  -nightly